dwave/plugins/torch/nn/functional.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2025 D-Wave
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Functional interface."""
+
+import torch
+
+__all__ = ["bit2spin_soft", "spin2bit_soft"]
+
+
+def bit2spin_soft(b: torch.Tensor) -> torch.Tensor:
+    """Maps input `b` to `2b-1`.
```

Comment:
Double back-ticks for code.
```suggestion
    """Maps input ``b`` to ``2b-1``.
```
Same for rest of docstring and file.

Reply:
Changed to (and other occurrences)
```
:math:`b`
```